cql3/statements/alter_keyspace_statement.hh

```diff
@@ -53,6 +53,8 @@ class query_processor;
 namespace statements {
 
 class alter_keyspace_statement : public schema_altering_statement {
+    static logging::logger _logger;
```

Comment:
nitpick: I think that loggers are better off being declared as static in .cc files. There's no reason to leak it to the header, especially that it's a private static member, so not really usable for anyone outside

Reply:
I agree, I just copied the code someone did in create_keyspace_statement.hh (in CREATE KEYSPACE, they already had code to log a warning, so they had to add a logger and to override execute(), and I now I needed exactly the same in ALTER KEYSPACE).
I'll change it (anyway this PR is stuck waiting for discussion on the format of the options ;-)).

Reply:
Done (I'll push a new version soon).
Although it's not really in the scope of the series, I'll change create_keyspace to do the same, just for symmetry.